include/SDL3/SDL_events.h

```diff
@@ -227,7 +227,11 @@ typedef enum
     /**
      *  This last event is only for bounding internal arrays
      */
-    SDL_EVENT_LAST    = 0xFFFF
+    SDL_EVENT_LAST    = 0xFFFF,
+
+    /* This just makes sure the enum is the size of Uint32 */
+    SDL_EVENT_ENUM_PADDING = 0x7FFFFFFF
```

Comment:
How did it pass `SDL_COMPILE_TIME_ASSERT(enum, sizeof(SDL_DUMMY_ENUM) == sizeof(int));` in SDL_stdinc.h before? Is this a platform with a 16 bit int?

Reply:
That assert doesn’t run on 3DS and Vita platforms. If we have an enumeration that needs to be a specific size, we need to do a little more work on it.